apps/glusterfs/app.go

```diff
@@ -93,9 +93,17 @@ type App struct {
 }
 
 // Use for tests only
```

Comment:
Can we please get rid of this line? This PR is the closest edit we have got. Pretty please :)

Reply:
I meant to and it slipped my mind. Thank you for catching that!!!